bitcoin/client.go

```diff
@@ -516,9 +516,9 @@ func (b *Client) parseTransactions(
 				"block hash", block.Hash,
 				"transaction hash", transaction.Hash,
 			)
-
+			status := SuccessStatus
 			for _, op := range txOps {
-				op.Status = SkippedStatus
+				op.Status = &status
```

Comment:
nit: the best way to do this is to do `types.String(SuccessStatus)` instead of creating a variable and then referencing it (should make these changes a little clearer)

Reply:
On this change in particular, it must be `SkippedStatus`, not `SuccessStatus`